extensions/eol-runtime-scanner/app.R

```diff
@@ -40,6 +39,27 @@ app_mode_lookup <- with(
   setNames(as.character(ind), values)
 )
 
+# Helper function to style versions based on server availability
+style_version_by_availability <- function(server_versions) {
+  function(value) {
+    value_str <- as.character(value)
+    if (!is.na(value_str)) {
+      # Check if version doesn't exist in server versions
+      if (!(value_str %in% server_versions)) {
+        return(list(
+          color = "#7D1A03", # Red color for versions not on server
+          fontWeight = "bold",
+          background = "#FFF0F0" # Light red background
+        ))
+      } else {
+        return(list(
+          color = "#276749" # Green color for versions on server
```

Comment:
Nice job on the colors here 👍 They are accessible which is tough for color-on-color text.

Reply:
I actually think that highlighting in color is perhaps a feature which will go away (it doesn't really communicate something easily understandable) but thanks 😂